README.md

```diff
@@ -168,6 +168,7 @@ We have several aladdin commands used for development and deployment. Note that
 - `--dev` mount host's aladdin directory onto aladdin container. Useful when developing aladdin.
 - `--skip-prompts` skip any confirmation messages during aladdin execution. Useful when automating commands.
 - `--non-terminal` run aladdin container without tty.
+- `--cluster-creating` skips cluster init commands such as namespace creation and helm init, useful for entering `aladdin bash` when the cluster is just being created
```

Comment:
Might it make sense instead to detect this?  Having a flag you have to know to pass to jump past certain things at an early stage seems a little awkward, particularly since you're telling aladdin something it already knows (that the cluster is creating).  If we do keep this, I might suggest naming it something that describes what it does (rather than what it's for), so that the caller can get a sense of the implications of using it (maybe `--unsafe`?  `--skip-init`?)